src/blocks/scratch3_sensing.js

```diff
@@ -99,4 +100,15 @@ Scratch3SensingBlocks.prototype.getKeyPressed = function (args, util) {
     return util.ioQuery('keyboard', 'getKeyIsDown', args.KEY_OPTION);
 };
 
+Scratch3SensingBlocks.prototype.daysSince2000 = function()
+{
+    var msPerDay = 24 * 60 * 60 * 1000;
+    var start = new Date(2000, 1-1, 1); 
+    var today =   new Date(); 
+    var dstAdjust = today.getTimezoneOffset() - start.getTimezoneOffset();
+    var mSecsSinceStart = today.valueOf() - start.valueOf();
+    mSecsSinceStart += ((today.getTimezoneOffset() - dstAdjust) * 60 * 1000);
+    return mSecsSinceStart / msPerDay;
```

Comment:
I know in Scratch 2.0 value is float, but I think here, in Scratch 3.0, it can be `Math.floor(...)`